dask_cloudprovider/aws/ecs.py

```diff
@@ -755,6 +768,16 @@ def __init__(
         self._worker_mem = worker_mem
         self._worker_gpu = worker_gpu
         self._worker_extra_args = worker_extra_args
+        self._worker_task_def_name = (
+            str(task_def_name) + "-worker-{uuid}"
+            if task_def_name
+            else "dask-worker-{uuid}"
+        ).format(uuid=str(uuid.uuid4())[:5])
+        self._scheduler_task_def_name = (
+            str(task_def_name) + "-scheduler-{uuid}"
+            if task_def_name
+            else "dask-scheduler-{uuid}"
+        ).format(uuid=str(uuid.uuid4())[:5])
```

Comment:
Instead of setting these values here could you just set them to whatever the kwarg is and then handle them in `_start`. If you have a look at all the other options they are done this way.

Then in `_start` you'll need to check if they are `None` and if so load them from `dask.config`. All default config values are set in `cloudprovider.yaml`.

Reply:
Sorry this hasn't been addressed yet. Marking as unresolved.